app/_reducers/localTransformReducer.ts

```diff
@@ -111,5 +135,18 @@ export default function localTransformReducer(
     case "align": {
       return align(action.line, action.to).mmul(localTransform);
     }
+    case "magnify": {
+      return scaleAboutPoint(action.scale, action.point).mmul(localTransform);
+    }
+    case "zoom_out": {
+      return fitPdfToView(action.layoutWidth, action.layoutHeight);
```

Comment:
Is there any chance we could show this in the center of the screen instead of all the way to the left? I had the layers menu open when I tried this, and it took a while for me to realize the zoomed out pages were hidden behind that menu
![Screenshot from 2024-06-19 20-52-41](https://github.com/Pattern-Projector/pattern-projector/assets/3938999/80300f7e-1826-48ec-997a-8098e15d60bc)

Reply:
Totally agree. I'll add it to the TODOs!